grails-app/views/wps/jobComplete.gsp

```diff
@@ -0,0 +1,11 @@
+Hi,
+
+Your IMOS download is ready. You can download it by clicking here: ${jobReportUrl}
```

Comment:
Yeah it does need to change, you're right.  I thought of that but forgot.  @pblain, what do you want the wording to be (this is the email that users get when the job has completed - be it successfully or not).

The other option is for the portal server to interrogate geoserver to find the status, but personally I would advise against this, at least for now, because it adds another point of failure (for a small amount of benefit).